src/groupByKey.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+var __assign = (this && this.__assign) || function () {
+    __assign = Object.assign || function(t) {
+        for (var s, i = 1, n = arguments.length; i < n; i++) {
+            s = arguments[i];
+            for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p))
+                t[p] = s[p];
+        }
+        return t;
+    };
+    return __assign.apply(this, arguments);
+};
+exports.__esModule = true;
+exports.groupByKey = void 0;
+function groupByKey(items, key) {
+    return items.reduce(function (groupedItems, item) {
+        var itemKey = String(item[key]);
+        var updatedGroupedItems = __assign({}, groupedItems);
+        if (itemKey in updatedGroupedItems) {
+            updatedGroupedItems[itemKey].push(item);
+        }
+        else {
+            updatedGroupedItems[itemKey] = [item];
+        }
+        return updatedGroupedItems;
+    }, {});
+}
+exports.groupByKey = groupByKey;
```

Comment:
Redundant, you need to write code only in `groupByKey.ts` file

Reply:
I got it, but I have some problems with the compiler and the versioning node. To compile my .ts file I need to delete my .js file and create a new compiler file again. 

(I specify the path to my .ts file in tsconfig.json and always run the command "tsc -p tsconfig.json --watch" but it doesn't work. It's probably a problem with my .js file because of that)